plugins/storage/volume/storpool/src/main/java/com/cloud/hypervisor/kvm/storage/StorPoolStorageAdaptor.java

```diff
@@ -149,6 +160,10 @@ public static String getVolumeNameFromPath(final String volumeUuid, boolean tild
     }
 
     public static boolean attachOrDetachVolume(String command, String type, String volumeUuid) {
+        if (volumeUuid == null) {
```

Comment:
command can be attach and detach only, right? can use bool instead of string, or for define any enum for operation and use that.